lib/semian/circuit_breaker.rb

```diff
@@ -51,14 +51,13 @@ def mark_failed(error)
 
     def mark_success
       return unless half_open?
-      @success_count += 1
+      push_time(@successes, @success_count_threshold)
```

Comment:
@byroot narh we don't really need to store the times of successes, a count is fine here. I just did it for consistency.